src/pages/Chat/Chat.styled.jsx

```diff
@@ -2,7 +2,7 @@ import styled from '@emotion/styled'
 import { bg, p, rounded, size, text } from '../../styles/mixins'
 
 export const ChatLayoutContainer = styled.div`
-  ${[ size({ width: '100vw', minHeight: '100vh' }), bg.white ]}
+${[ size({ width: '100vw', minHeight: '100vh' }), bg.white ]}
```

Comment:
check indentation

Reply:
thanks, I hadn't noticed that.
It is fixed now.